addons/website_livechat/static/tests/tours/website_livechat_chatbot_flow.js

```diff
@@ -1,200 +1,260 @@
-            trigger: messagesContain("Restarting conversation..."),
-        },
-        {
-            // check first welcome message is posted
-            trigger: messagesContain("Hello! I'm a bot!"),
-        },
-        {
-            // check second welcome message is posted
-            trigger: messagesContain("I help lost visitors find their way."),
-        },
-        {
-            // check question_selection message is posted
-            trigger: messagesContain("How can I help you?"),
-        },
-        {
-            trigger: '.o-livechat-root:shadow li:contains("Pricing Question")',
-            run: "click",
-        },
-        {
-            // the path should now go towards 'Pricing Question (first part)'
-            trigger: messagesContain(
-                "For any pricing question, feel free ton contact us at pricing@mycompany.com"
-            ),
-        },
-        {
-            // the path should now go towards 'Pricing Question (second part)'
-            trigger: messagesContain("We will reach back to you as soon as we can!"),
-        },
-        {
-            // should ask for website now
-            trigger: messagesContain("Would you mind providing your website address?"),
-        },
-        {
-            trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
-            run: "edit no",
-        },
-        {
-            trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
-            run: "press Enter",
-        },
-        {
-            // should ask for feedback now
-            trigger: messagesContain("Great, do you want to leave any feedback for us to improve?"),
-        },
-        {
-            trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
-            run: "edit no, nothing so say",
-        },
-        {
-            trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
-            run: "press Enter",
-        },
-        {
-            trigger: messagesContain("Ok bye!"),
-            run: "click",
-        },
-        {
-            trigger:
-                ".o-livechat-root:shadow .o-mail-ChatWindow-command[title='Restart Conversation']",
-            run: "click",
-        },
-        {
-            trigger: ".o-livechat-root:shadow li:contains(I want to speak with an operator)",
-            run: "click",
-        },
-        {
-            trigger: messagesContain("I will transfer you to a human."),
-        },
-        {
-            trigger: ".o-livechat-root:shadow .o-mail-Composer-input:enabled",
-        },
-    ],
+    steps: () => {
+        patchWithCleanup(window, {
+            debounceAnswerCount: 0,
+        });
+        patchWithCleanup(odoo.__WOWL_DEBUG__.root.env.services["mail.store"].Chatbot.prototype, {
+            async _processMultiLineAnswer(message) {
+                window.debounceAnswerCount++;
+                return await super._processMultiLineAnswer(message);
+            },
+        });
```

Comment:
Put brief comment to motivate the intent of patch: multi-line answer termination is dictate by some delay when user hasn't typed anything. Counting a function call is heuristics to detect chatbot is checking regularly when the visitor has finished providing the multi-line answer.